packages/frontend/src/lib/ModelSelect.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+import { faCheckCircle, faDownload } from '@fortawesome/free-solid-svg-icons';
+import Select from 'svelte-select';
+import Fa from 'svelte-fa';
+import type { ModelInfo } from '@shared/src/models/IModelInfo';
+
+export let disabled: boolean = false;
+/**
+ * Recommended model ids
+ */
+export let recommended: string[] | undefined = undefined;
+
+/**
+ * List of models
+ */
+export let models: ModelInfo[];
```

Comment:
for new components, should we leverage svelte5 or not ?

like
let {disabled, recommended....... = $props();

and use runes rather than `$:`

Reply:
I am looking at the experimental documentation, and I don't find a nice a way to type the `$props();` 

https://component-party.dev/?f=svelte4,svelte5#component-composition.props